### ISSUE TITLE
Id overwrites settings when editing

### ISSUE THREAD

**Issue by utrescu** [SilentVoid13/Chesser#15]
**Plugin informations (please complete the following information):**
 - OS: Linux
 - Chesser version: 0.2.1
 - Obsidian version: 1.0.3

**Describe the bug**

Unable to edit settings of a existing board because it's overwrited by `id` setting

ex. I tried to write `fen`:
![writing fen](https://github.com/XavierSala/Images/blob/9c0b2870f3ea4b510f16d8b0711a9171778c50a4/chesser2.gif?raw=true)

**Expected behavior**
I expect to edit settings without interferences

**Screenshots**

**Additional context**

**Comment by dhniceday** [comment]
Same behavior on my end. Makes plugin unusable.

**Comment by dhniceday** [comment]
Yes, here also the same  on Fedora 40. Plugin is unusable in this state.
